src/Styleguide/Elements/__stories__/Typography.story.tsx

```diff
@@ -122,15 +122,28 @@ stories.add("Typography", () => {
           </a>
         </Display>
       </Section>
-      <Section title="Links .colorLink">
+      <Section title="Links .purpleLink">
         <Sans mx={5}>
-          <a href="#" className="colorLink">
+          <a href="#" className="purpleLink">
             This is a link
           </a>
         </Sans>
 
         <Serif mx={5}>
-          <a href="#" className="colorLink">
+          <a href="#" className="purpleLink">
+            This is a link
+          </a>
+        </Serif>
+      </Section>
+      <Section title="Links .blackLink">
```

Comment:
In our case the Artist name is a Serif link but black, so added another option here and renamed the original colored one into more explicit purpleLink. Hope it's ok @damassi

Reply:
@oxaudo - It defaults to black, so no need to add an additional class. Re `purpleLink`Its ok with me but @briansw should chime in as I'm following the link spec precisely (meaning, following naming). If `colorLink` is meant to refer specifically to purple (and thus eliminate the possibility of someone arbitrarily introducing a new color class) then we should keep things aligned, but if `colorLink` means "pass in a color" then that's something else. 

@briansw - thoughts?

Reply:
presently we'd never make a link any color but purple.

Reply:
@zephraph - this also addresses our discussion about `<Link>` primitives. Less options is more (so to speak)

Reply:
The default is actually `color: ${themeGet("colors.black60")};` which makes it gray. That is why I added this black color.

Reply:
@oxaudo - I see what you mean now: 

![screen shot 2018-06-11 at 1 34 42 pm](https://user-images.githubusercontent.com/236943/41255722-57826c72-6d7c-11e8-9626-0028fe075cad.png)

Reply:
So this is my mistake, rather than assigning a color it should be `color: inherit` (contextual). I was specifically assigning a color to the link.

Reply:
I'll revert the other change then and let it inherit the color from parent. Thanks for checking here!